cmake/find/Finddoctest.cmake

```diff
@@ -0,0 +1,20 @@
+# Copyright Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# This module is an opt-in find module _only_ for developers of the SDK who
+# wish not to use vcpkg. To tell CMake about this find module, pass this file's
```

Comment:
Let's use "cannot use", as oppose to "wish not to use"

Reply:
Did you want me to change that for the FindGSL.cmake as well?

Reply:
> Did you want me to change that for the FindGSL.cmake as well?

Sounds good.  Thanks!